config/webpack.js

```diff
@@ -162,7 +162,8 @@ module.exports = (env) => {
               loader: 'css-loader',
               options: {
                 modules: {
-                  localIdentName: env.dev ? '[name]_[local]-[hash:base64:5]' : '[hash:base64:5]'
+                  localIdentName:
+                    env.dev || env.beta ? '[name]_[local]-[hash:base64:5]' : '[hash:base64:5]'
```

Comment:
@sundevour I also changed the build so beta will have more meaningful CSS class names.